staging/src/k8s.io/client-go/util/workqueue/metrics_test.go

```diff
@@ -279,3 +280,35 @@ func TestMetrics(t *testing.T) {
 		t.Errorf("expected %v, got %v", e, a)
 	}
 }
+
+func TestLeak(t *testing.T) {
+	defer func() {
+		if err := goleak.Find(goleak.IgnoreTopFunction("k8s.io/client-go/util/workqueue.(*delayingType[...]).waitingLoop")); err != nil {
```

Comment:
I don't think you need to ignore `delayingType`. AFAIU `newQueueWithConfig` is not delaying queue.